### PR TITLE
Issue refreshing the page creates new api token

### PR DESCRIPTION
**Pull request by dinesh-skyach**
Issue refreshing the page creates new api token

### CODE REVIEW

File: app/views/private/api_tokens/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<p><%= t('.desc') %></p>
```
Comment (yivo) [inline app/views/private/api_tokens/show.html.erb]: @dinesh-skyach I see this code exists in another file. Why it is duplicated?

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by dinesh-skyach** [comment]
Hi @yivo  , Sorry about that, instead of avoiding duplicacy, we needed to remove the create.html.erb and shift its code to show.html.erb as create.html.erb should not have been there if we follow rails way. I missed to remove the file earlier. Please review.

**Comment by yivo** [comment]
@dinesh-skyach Thanks. We will assign our QA to review it.

**Comment by yivo** [comment]
ping @mkuvshynnikov

**Review by yivo — APPROVED** [review]

**Comment by yivo** [comment]
ping @mkuvshynnikov

**Pull request merged**